wallet/wallet.c

```diff
@@ -283,6 +284,13 @@ static const struct utxo **wallet_select(const tal_t *ctx, struct wallet *w,
 		struct amount_sat needed;
 		struct utxo *u = tal_steal(utxos, available[i]);
 
+		/* If we require confirmations check that we have a
+		 * confirmation height and that it is below the required
+		 * maxheight (current_height - minconf */
+		if (maxheight != 0 &&
+		    (!u->blockheight || *u->blockheight > maxheight))
```

Comment:
tbh i find `maxheight` not intuitive wrt to what you're checking for here. maybe something like `gated_confirmation_height` or `req_conf_height` ... neither of these seems exactly right.

Reply:
Yeah, I've been going back and forth with the name, it's the maximum acceptable `confirmation_height` really, and being maybe the special value `0` is what makes it confusing.